### DISCUSSION TITLE
Does the Payload Website Use Strapi?

### DISCUSSION THREAD

**Discussion by nimazabi** [payloadcms/payload#9827] [General]
As someone new to Payload, I’m really excited to explore how it can serve as a reliable headless CMS for large-scale, real-world projects. While checking out website technologies using tools like Wappalyzer, I noticed that the Payload website itself is using Strapi as its CMS instead of its own technology. Could you clarify why this is the case? Sending love and appreciation to the Payload community!

**Comment by akhrarovsaid** [comment]
Hey @nimazb,

The Payload website does not use Strapi - it uses Payload. It's open source so you can easily have a look for yourself [right here](https://github.com/payloadcms/website?tab=readme-ov-file).

**Comment by denolfe** [comment]
Interestingly enough, I use Wappalyzer, and it was indeed detecting Strapi (which has obviously never been the case).

I'll be submitting something to the extension.
